.github/workflows/upstream-dev-ci.yaml

```
@@ -43,6 +45,11 @@ jobs:
         run: |
           set -o pipefail
           python -m pytest -rf | tee output-${{ matrix.python-version }}-log
+      - name: Set availability status of logs
+        if: failure()
+        run: |
+          echo '::set-output name=ARTIFACTS_AVAILABLE::true'
+          exit 1
```

we should be able to do this in the step above:
```suggestion
          python -m pytest -rf | tee output-${{ matrix.python-version }}-log
          export status="$?"
          [ "$status" != 0 ] && echo '::set-output name=ARTIFACTS_AVAILABLE::true'
          exit "$status"
```
but I'm not sure saving a step is actually necessary.

---

I just tested this on a toy repo, and for some reason it's not working as expected. So, I'm keeping the two steps for the time being...

---

that's a good idea, I'll play around with this, too